pkgs/development/node-packages/default-v10.nix

```diff
@@ -5,8 +5,12 @@ let
     inherit pkgs nodejs;
     inherit (stdenv.hostPlatform) system;
   };
+  nodeSourcePackages = import ./composition-source-v10.nix {
```

Comment:
IIRC it's a standard convention to create such a big package in its own directory rather than adding it to `development/node-packages`.

Reply:
Package moved to its own directory.